Makefile

```diff
@@ -1,38 +1,34 @@
+## DEVELOPMENT
+.PHONY: build
+build: 
+	@go build
 
-TAG=$(shell git describe --abbrev=0 --tags)
-
-NAME=conftest
-IMAGE=openpolicyagent/$(NAME)
-ALT_IMAGE=instrumenta/$(NAME)
```

Comment:
I think we should keep these for backward compatibility reasons, at least until we can formally deprecate and communicate that effectively.

Reply:
Makes sense to me, no objections here!